README.md

```diff
@@ -10,6 +10,7 @@ like `fetch` and `clone`, and to validate the usability and control of the API o
 
 `gitoxide` aspires to be a production-grade server implementation and the `ein` binary aspires to become the default way to interact with git repositories.
 
+[![asciicast](https://asciinema.org/a/542159.svg)](https://asciinema.org/a/542159)
```

Comment:
```suggestion
```

Reply:
Sorry for adding the line. Should have looked at the change again 😅

Reply:
No worries, that's what I am here for :).